src/lib/components/ApplicationDetail/index.svelte

```diff
@@ -1,13 +1,45 @@
 <script lang="ts">
+	import { get } from 'svelte/store';
+	import { providerStore } from 'web3-stores';
+	import { workstreamStore } from '$lib/stores/workstreamsStore.js';
+	const workstreams = get(workstreamStore);
```

Comment:
In a svelte component you should be able to subscribe to the store directly by using the `$` sign.

`$workstreamStore`

Reply:
Good point!